_notices/rdn0003.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: notice
+parent: RAPIDS Developer Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rdn
+# Update meta-data for notice
+notice_id: 3 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Devel Container Release Change"
+notice_author: RAPIDS Ops
+notice_status: Completed
+notice_status_color: green
```

Comment:
should this be yellow since 21.12 is in the process of being released and the nightly packages for CUDA 11.5 are not currently available?

Reply:
Good catch. I think since we've stopped building the `21.12` non-`11.5` nightlies at this point, it's probably fine to leave it as `green`/`Completed`.